packages/js/tracks/src/stats.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import debug from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import { isDevelopmentMode } from './utils';
+
+/**
+ * Module variables
+ */
+const tracksDebug = debug( 'wc-admin:tracks:stats' );
+
+/**
+ * Builds a querystring from the given group and name parameters.
+ *
+ * This will automatically add the prefix `x_woocommerce-` to the group name.
+ *
+ * @param {Record<string, string> | string} group  - The group of stats or a single stat name.
+ * @param {string}                          [name] - The name of the stat if group is a string.
+ * @return {string} The constructed querystring.
+ */
+function buildQuerystring(
+	group: Record< string, string > | string,
+	name?: string
+): string {
+	let uriComponent = '';
```

Comment:
```suggestion
const PREFIX = 'x_woocommerce-'
 if (typeof group !== object) { return (new URLSearchParams( { `${PREFIX}${group}` : name } )).toString();
 else {
  const prefixedGroups = {};
  Object.entries(group).forEach(([key, value]) => { 
    prefixedGroups[`${PREFIX}${key}`] = value;
   })
   return (new URLSearchParams(prefixedGroups)).toString();
 }

```

I wrote this in the comment box so it's not 100% tested but I think this should work!

Reply:
This is a great suggestion! @rjchow  I've made the changes based on your feedback. 🙌  f858ff5ed095b4f72b722b6af5ca67e393c3c2d0